website/blog/2019-08-21-announcing-riff-0-4-0.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Announcing riff v0.4.0"
+---
+
+We are happy to announce the release of riff v0.4.0. Thank you all riff, Cloud Native Buildpack, and Knative contributors. A lot has changed in this release; we're excited about having a new foundation to take riff to 1.0.
+
+A lot has changed, buckle up.
+
+<!--truncate-->
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/cli/releases/tag/v0.4.0) on GitHub. Please follow one of the [getting started](/docs/v0.4/getting-started) guides, to create a new cluster on GKE, Minikube, Docker Desktop for Mac, or Docker Desktop for Windows.
+
+## Notable changes
+
+### Knative and Istio
+
+riff no longer has a hard dependency on Knative and Istio. Knative integration is provided by riff's [Knative runtime](#knative-runtime), which is available in clusters with Knative Serving installed. Istio is only required for the Knative runtime.
+
+### builds
+
+riff Builds now encompass more than just functions with the addition of application and container builds. Each build resource reports that latest image that a [runtime](#runtimes) uses to deploy the built workload. The built image is always reported as a digested image to make the detection and rollout of changes more deterministic.
+
+Builds are transitioning from a one-off tasks to a continuous stream of images. riff 0.4 sets up the API to enable ongoing updated builds, but does not yet hook into sources to trigger rebuilding.
```

Comment:
"Builds are transitioning from one-off tasks to continuous streams of images." (removed "a" 2x and added "s" to stream)